src/minesweeper/solver/GameRobot.java

```diff
@@ -144,6 +153,25 @@ public boolean unCoverCell(int row, int col) {
         return response;
     }
 
+    public boolean random_try() {
+        // this function performs random try on a covered cell 
+        // this strategy works well in (60,400)
+        int done = 0;
+        while(done == 0){
+            int randomKey = random.nextInt(N * N);
+            int row = randomKey / N;
+            int col = randomKey % N;
+            CellState s = board.getCellState(row, col);
+            if(CellState.isUncovered(s) == false){
```

Comment:
The state check here is incorrect. Check for Covered cells only.

Reply:
Thanks for reminding me. Fixed it.